webpack/assets/javascripts/react_app/components/PF4/InlineEdit/InlineEdit.stories.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import { withKnobs, text, boolean } from '@theforeman/stories';
+import Story from '../../../../../../stories/components/Story';
+import InlineEdit from './InlineEdit';
+
+export default {
+  title: 'Components|PF4/InlineEdit',
+  decorators: [withKnobs],
+};
+
+// eslint complains about hooks in storybooks
+export const defaultStory = () => {
+  const [value, setValue] = useState('hello');
+  const [value2, setValue2] = useState('hello again');
+
+  const onEdit = async (v, a) => {
+    // delay to emulate an asynchronous backend call
+    await (async () => new Promise(resolve => setTimeout(resolve, 1000)))();
+    setValue(v);
+    alert(`Update of ${a} successful`);
+  };
+
+  const onEdit2 = async (v, a) => {
+    // delay to emulate an asynchronous backend call
+    await (async () => new Promise(resolve => setTimeout(resolve, 1000)))();
+    setValue2(v);
+    alert(`Update of ${a} successful`);
+  };
+
+  return (
+    <Story>
+      <InlineEdit
+        value={text('value', value)}
+        attribute="backend_identifier"
+        textArea={boolean('text area', false)}
+        onEdit={onEdit}
+      />
+      <InlineEdit
+        value={value2}
+        attribute="backend_identifier2"
+        onEdit={onEdit2}
+      />
+    </Story>
+  );
+};
+
+export const withTextArea = () => {
+  const [value, setValue] = useState('hello');
```

Comment:
For the text area story it would be good to document the enter key feature here by setting the default value.
```suggestion
  const [value, setValue] = useState('Text area: Pressing enter will not submit');
```